core/domain/exp_domain.py

```diff
@@ -2154,20 +2174,36 @@ def extract_content_id_from_choices(html: str) -> str:
                 # The solution type will be migrated from SetOfHtmlString to
                 # SetOfTranslatableHtmlContentIds.
                 if solution is not None:
+                    # Ruling out the possibility of Any other type for mypy type
```

Comment:
Any --> any, ditto elsewhere. (You are using "any" in the colloquial sense here and not as the type "Any".)